core/domain/subscription_services.py

```diff
@@ -206,6 +268,11 @@ def record_user_has_seen_notifications(user_id, last_seen_msecs):
     """Updates the last_checked time for this user (which represents the time
     the user last saw the notifications in the dashboard page or the
     notifications dropdown).
+
+    Args:
+        user_id: str. The user ID of the new subscriber.
+        last_seen_msecs: str. The time (Last seen) when the user last saw the
```

Comment:
I don't think you need the part in parentheses.

Do you want to specify "milliseconds since the Epoch"?

Reply:
Done! I have added "msecs since the Epoch".